osu.Game/Graphics/UserInterface/ExternalLinkButton.cs

```diff
@@ -42,6 +49,31 @@ public ExternalLinkButton(string link = null)
             };
         }
 
+        private void copyUrl()
+        {
+            host.GetClipboard()?.SetText(Link);
+            notificationOverlay.Post(new SimpleNotification
+            {
+                Text = "Copied URL!"
+            });
```

Comment:
A notification shouldn't be shown when copying a URL, that would only be annoying to the user.